Moki/client/src/js/dashboards/Connectivity/ConnectivityCharts.js

```diff
@@ -8,7 +8,7 @@ import TopologyChart from '../../charts/topology_chart.js';
 import Heatmap from '../../charts/heatmap_chart.js';
 import store from "../../store/index";
 import LoadingScreenCharts from '../../helpers/LoadingScreenCharts';
-import {parseHeatmapData, parseDateHeatmap, parseTopologyData} from '@moki-client/es-response-parser';
+import {parseHeatmapDataDecrypt, parseDateHeatmapDecrypt, parseTopologyDataDecrypt} from '@moki-client/es-response-parser';
```

Comment:
@Akaren13 how are we going to maintain public version vs. private version?
what are the *Decrypt versions going to do in the public version?